packages/flame/test/components/collidable_type_test.dart

```diff
@@ -8,8 +8,9 @@ class _HasCollidablesGame extends FlameGame with HasCollidables {}
 
 class _TestBlock extends PositionComponent with HasHitboxes, Collidable {
   final List<Collidable> collisions = List.empty(growable: true);
+  String? name;
 
-  _TestBlock(Vector2 position, Vector2 size, CollidableType type)
+  _TestBlock(Vector2 position, Vector2 size, CollidableType type, {this.name})
```

Comment:
Why was this changed in this PR?

Reply:
For testing purposes: I was getting error messages that `Instance of _TestBlock` was not equal to an `Instance of _TestBlock`, and that just wasn't very helpful. This doesn't affect the nature of the test, only what message it emits when it fails.